app/views/people/show_tasks.html.erb

```diff
@@ -0,0 +1,43 @@
+<h3>Incomplete Tasks:</h3>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Date Created</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @person.tasks.where(date_completed: nil).each do |task| %>
```

Comment:
This works here - but you could also set two instance variables in your controller action that already set them as complete or incomplete